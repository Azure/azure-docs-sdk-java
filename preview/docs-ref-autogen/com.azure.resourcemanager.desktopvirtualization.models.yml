### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.desktopvirtualization.models"
fullName: "com.azure.resourcemanager.desktopvirtualization.models"
name: "com.azure.resourcemanager.desktopvirtualization.models"
summary: "Package containing the data models for DesktopVirtualizationApiClient. null."
classes:
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroupList"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroupPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroupType"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationList"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationType"
- "com.azure.resourcemanager.desktopvirtualization.models.CommandLineSetting"
- "com.azure.resourcemanager.desktopvirtualization.models.DesktopList"
- "com.azure.resourcemanager.desktopvirtualization.models.DesktopPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.ExpandMsixImageList"
- "com.azure.resourcemanager.desktopvirtualization.models.HealthCheckName"
- "com.azure.resourcemanager.desktopvirtualization.models.HealthCheckResult"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPoolList"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPoolPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPoolType"
- "com.azure.resourcemanager.desktopvirtualization.models.Identity"
- "com.azure.resourcemanager.desktopvirtualization.models.LoadBalancerType"
- "com.azure.resourcemanager.desktopvirtualization.models.LogSpecification"
- "com.azure.resourcemanager.desktopvirtualization.models.MigrationRequestProperties"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixImageUri"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackageApplications"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackageDependencies"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackageList"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackagePatch"
- "com.azure.resourcemanager.desktopvirtualization.models.Operation"
- "com.azure.resourcemanager.desktopvirtualization.models.OperationProperties"
- "com.azure.resourcemanager.desktopvirtualization.models.PersonalDesktopAssignmentType"
- "com.azure.resourcemanager.desktopvirtualization.models.Plan"
- "com.azure.resourcemanager.desktopvirtualization.models.PreferredAppGroupType"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpoint"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointConnectionListResultWithSystemData"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.desktopvirtualization.models.PublicNetworkAccess"
- "com.azure.resourcemanager.desktopvirtualization.models.RegistrationInfoPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.RegistrationTokenOperation"
- "com.azure.resourcemanager.desktopvirtualization.models.RemoteApplicationType"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceModelWithAllowedPropertySet"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceModelWithAllowedPropertySetIdentity"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceModelWithAllowedPropertySetPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceModelWithAllowedPropertySetSku"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceProviderOperationDisplay"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceProviderOperationList"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingHostPoolReference"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingHostPoolType"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlanList"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlanPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingSchedule"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingScheduleDaysOfWeekItem"
- "com.azure.resourcemanager.desktopvirtualization.models.SendMessage"
- "com.azure.resourcemanager.desktopvirtualization.models.ServiceSpecification"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHostHealthCheckFailureDetails"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHostHealthCheckReport"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHostList"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHostLoadBalancingAlgorithm"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHostPatch"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionState"
- "com.azure.resourcemanager.desktopvirtualization.models.Sku"
- "com.azure.resourcemanager.desktopvirtualization.models.SsoSecretType"
- "com.azure.resourcemanager.desktopvirtualization.models.StartMenuItemList"
- "com.azure.resourcemanager.desktopvirtualization.models.Status"
- "com.azure.resourcemanager.desktopvirtualization.models.StopHostsWhen"
- "com.azure.resourcemanager.desktopvirtualization.models.Time"
- "com.azure.resourcemanager.desktopvirtualization.models.UpdateState"
- "com.azure.resourcemanager.desktopvirtualization.models.UserSessionList"
- "com.azure.resourcemanager.desktopvirtualization.models.WorkspaceList"
- "com.azure.resourcemanager.desktopvirtualization.models.WorkspacePatch"
enums:
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceIdentityType"
- "com.azure.resourcemanager.desktopvirtualization.models.SkuTier"
interfaces:
- "com.azure.resourcemanager.desktopvirtualization.models.Application"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithApplicationType"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithCommandLineArguments"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithCommandLineSetting"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithFilePath"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithIconIndex"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithIconPath"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithMsixPackageApplicationId"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithMsixPackageFamilyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.DefinitionStages.WithShowInPortal"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithApplicationType"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithCommandLineArguments"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithCommandLineSetting"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithFilePath"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithIconIndex"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithIconPath"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithMsixPackageApplicationId"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithMsixPackageFamilyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithShowInPortal"
- "com.azure.resourcemanager.desktopvirtualization.models.Application.UpdateStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithApplicationGroupType"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithHostPoolArmPath"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithKind"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithManagedBy"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithMigrationRequest"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithSku"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.DefinitionStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.UpdateStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroup.UpdateStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.ApplicationGroups"
- "com.azure.resourcemanager.desktopvirtualization.models.Applications"
- "com.azure.resourcemanager.desktopvirtualization.models.Desktop"
- "com.azure.resourcemanager.desktopvirtualization.models.Desktops"
- "com.azure.resourcemanager.desktopvirtualization.models.ExpandMsixImage"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithCustomRdpProperty"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithHostPoolType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithKind"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithLoadBalancerType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithManagedBy"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithMaxSessionLimit"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithMigrationRequest"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithPersonalDesktopAssignmentType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithPreferredAppGroupType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithRegistrationInfo"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithRing"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithSku"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithSsoClientId"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithSsoClientSecretKeyVaultPath"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithSsoSecretType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithSsoadfsAuthority"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithStartVMOnConnect"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithValidationEnvironment"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.DefinitionStages.WithVmTemplate"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithCustomRdpProperty"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithLoadBalancerType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithMaxSessionLimit"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithPersonalDesktopAssignmentType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithPreferredAppGroupType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithRegistrationInfo"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithRing"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithSsoClientId"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithSsoClientSecretKeyVaultPath"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithSsoSecretType"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithSsoadfsAuthority"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithStartVMOnConnect"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithValidationEnvironment"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPool.UpdateStages.WithVmTemplate"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPoolProperties"
- "com.azure.resourcemanager.desktopvirtualization.models.HostPools"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixImages"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithDisplayName"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithImagePath"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithIsActive"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithIsRegularRegistration"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithLastUpdated"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithPackageApplications"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithPackageDependencies"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithPackageFamilyName"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithPackageName"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithPackageRelativePath"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.DefinitionStages.WithVersion"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.UpdateStages.WithDisplayName"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.UpdateStages.WithIsActive"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackage.UpdateStages.WithIsRegularRegistration"
- "com.azure.resourcemanager.desktopvirtualization.models.MsixPackages"
- "com.azure.resourcemanager.desktopvirtualization.models.Operations"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointConnectionWithSystemData"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateLinkResource"
- "com.azure.resourcemanager.desktopvirtualization.models.PrivateLinkResources"
- "com.azure.resourcemanager.desktopvirtualization.models.RegistrationInfo"
- "com.azure.resourcemanager.desktopvirtualization.models.ResourceProviderOperation"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithExclusionTag"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithHostPoolReferences"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithHostPoolType"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithKind"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithManagedBy"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithSchedules"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithSku"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.DefinitionStages.WithTimeZone"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithExclusionTag"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithHostPoolReferences"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithSchedules"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlan.UpdateStages.WithTimeZone"
- "com.azure.resourcemanager.desktopvirtualization.models.ScalingPlans"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHost"
- "com.azure.resourcemanager.desktopvirtualization.models.SessionHosts"
- "com.azure.resourcemanager.desktopvirtualization.models.StartMenuItem"
- "com.azure.resourcemanager.desktopvirtualization.models.StartMenuItems"
- "com.azure.resourcemanager.desktopvirtualization.models.UserSession"
- "com.azure.resourcemanager.desktopvirtualization.models.UserSessions"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.Definition"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.Blank"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithApplicationGroupReferences"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithKind"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithManagedBy"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithPlan"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithSku"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.DefinitionStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.Update"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages.WithApplicationGroupReferences"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages.WithDescription"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages.WithFriendlyName"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspace.UpdateStages.WithTags"
- "com.azure.resourcemanager.desktopvirtualization.models.Workspaces"
metadata: {}
package: "com.azure.resourcemanager.desktopvirtualization.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-desktopvirtualization:1.0.0-beta.1
