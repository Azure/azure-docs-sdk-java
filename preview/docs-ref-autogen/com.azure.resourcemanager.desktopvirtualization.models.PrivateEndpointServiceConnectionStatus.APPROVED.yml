### YamlMime:JavaMember
uid: "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.desktopvirtualization.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.desktopvirtualization.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-desktopvirtualization:1.0.0-beta.1
